book.json

```
@@ -19,6 +19,7 @@
       "url": "https://github.com/commercetools/nodejs/"
     },
     "ga": {
+      "anonymizeIp": true,
```

From what I understand it should be like this:

```suggestion
      "configuration": {
        "anonymizeIp": true,
      },
```

https://www.npmjs.com/package/gitbook-plugin-ga